charts/stable/minecraft-java/values.yaml

```diff
@@ -99,6 +92,9 @@ workload:
                 - mc-health
           env:
             SERVER_PORT: "{{ .Values.service.main.ports.main.port }}"
+            QUERY_PORT: "{{ .Values.service.main.ports.main.query }}"
```

Comment:
`main.query`?
Is this supposed to be an extra port? maybe `ports.query.port`?

Reply:
Is it possible to add to main a query port so main uses tcp 25565 and query uses udp 25565.

Or... should it fully be a separate port all together?

Reply:
If tcp and udp are 2 diff things, create new service .
if its for the same purpsoe, add a port to main